src/renderers/testing/ReactTestRenderer.js

```diff
@@ -83,8 +86,11 @@ class ReactTestComponent {
 
   getPublicInstance(transaction: ReactTestReconcileTransaction): Object {
```

Comment:
Can we remove transaction as an argument here now? As well as not pass it wherever we currently do?